falcon/middleware.py

```diff
@@ -43,22 +43,32 @@ class CORSMiddleware(UniversalMiddlewareWithProcessResponse):
             (default ``None``).
 
             See also:
-            https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers
+                * https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers
         allow_credentials (Optional[Union[str, Iterable[str]]]): List of origins
             (case sensitive) for which to allow credentials via the
             ``Access-Control-Allow-Credentials`` header.
             The string ``'*'`` acts as a wildcard, matching every allowed origin,
             while ``None`` disallows all origins. This parameter takes effect only
             if the origin is allowed by the ``allow_origins`` argument.
-            (Default ``None``).
+            (default ``None``).
+        allow_private_network (bool):
+            If ``True``, the server includes the
+            ``Access-Control-Allow-Private-Network`` header in responses to
+            CORS preflight (OPTIONS) requests. This indicates that the resource is
+            willing to respond to requests from less-public IP address spaces
+            (e.g., from public site to private device).
+            (default ``False``).
 
+            See also:
+                * https://wicg.github.io/private-network-access/#private-network-request-heading
```

Comment:
Same here, this doesn't need to be a bullet list. I think the line was just broken in the case above just because the link is very long, but Sphinx still parses it as one line.